code/modules/projectiles/projectile.dm

```diff
@@ -199,11 +199,16 @@
 		L.on_hit(src)
 
 	var/reagent_note
-	if(reagents?.reagent_list)
+	if(length(reagents?.reagent_list) > 0)
```

Comment:
inb4 someone asks, for some reason this projectile has reagents.reagent_list with the length of 0 and it got logged as just "REAGENTS:" so it prevents double logging.